.github/workflows/gh-build-and-test.yml

```diff
@@ -36,3 +36,23 @@ jobs:
       upload-enabled: ${{ inputs.upload-enabled }}
       python-version: ${{ inputs.python-version }}
     secrets: inherit
+
+  test:
+    if: ${{ github.repository_owner == 'nvidia' }}
+    needs:
+      - build
+    uses:
+      ./.github/workflows/gh-test.yml
+    with:
+      client-repo: ${{ github.event.repository.name }}
+      target-device: ${{ inputs.target-device }}
+      test-options: ${{ inputs.build-type }}
+      runs-on: ${{ (inputs.host-platform == 'linux-x64' && 'linux-amd64-2gpu') || (inputs.host-platform == 'linux-aarch64' && 'linux-arm64-cpu16') || (inputs.host-platform == 'mac' && 'macos-latest') }}
```

Comment:
Q: @sandeepd-nv I don't see the `linux-amd64-2gpu` runner in the internal dashboard? Maybe this is why the test stage hangs?